### PR TITLE
Login template implemented

### PR DESCRIPTION
**Pull request by luksdlt92**
## Summary

Login template implemented.

```WoloxLoginFragment``` needs a ```LoginConfiguration``` to init. The ```LoginField``` is an enum that represents the possible fields inside the login.

In ```LoginConfiguration```, you must set:
* Layout id that will be used to populate the fragment
* Username and password edit text ids (using ```LoginField``` enum)
* Username and password ```validators``` (like email validator or min/max length for password). A ```validator``` is only used for EditText for now
* That's it! Just instance the fragment, set the configuration and replace it into the activity

Also, you should set:
* Other views like Facebook, Forgot password, signup
* Custom OnClick and OnLongClick listeners in any view represented by ```LoginField```

## Trello card

https://trello.com/c/3ccnv7B0/8-validadores

### CODE REVIEW

File: authentication/src/main/java/ar/com/wolox/wolmo/authentication/fragment/WoloxLoginFragment.java [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+                if (!checkValidators(LoginField.USERNAME, username, true)
+                        || !checkValidators(LoginField.PASSWORD, password, true)) {
+                    return;
+                }
+
+                mPresenter.login(username, password);
+            }
+        });
+
+        mUsername.addTextChangedListener(new TextWatcher() {
```
Comment (framundo) [inline authentication/src/main/java/ar/com/wolox/wolmo/authentication/fragment/WoloxLoginFragment.java]: Can we implement an abstract TextWatcher which does nothing on `afterTextChanged` and `beforeTextChanged`? This way we can clean up this code to a more readable one. Maybe `DummyTextWatcher` or `EmptyTextWatcher` are suitable names for it?

**Comment by framundo** [comment]
@luksdlt92  Great work, really

File: authentication/src/main/java/ar/com/wolox/wolmo/authentication/fragment/WoloxLoginFragment.java (hunk 1 above)
Comment (luksdlt92) [inline authentication/src/main/java/ar/com/wolox/wolmo/authentication/fragment/WoloxLoginFragment.java]: Sure.

**Comment by framundo** [comment]
:+1: Merge this

**Pull request merged**